pkg/cmd/exoscale.go

```diff
@@ -2,29 +2,31 @@ package cmd
 
 import (
 	"fmt"
+	"time"
 
 	"github.com/urfave/cli/v2"
-	"github.com/vshn/exoscale-metrics-collector/pkg/clients/cluster"
-	"github.com/vshn/exoscale-metrics-collector/pkg/clients/exoscale"
-	"github.com/vshn/exoscale-metrics-collector/pkg/exoscale/dbaas"
-	"github.com/vshn/exoscale-metrics-collector/pkg/exoscale/sos"
+	"github.com/vshn/exoscale-metrics-collector/pkg/exoscale"
+	"github.com/vshn/exoscale-metrics-collector/pkg/kubernetes"
 	"github.com/vshn/exoscale-metrics-collector/pkg/log"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+// billingHour represents the hour when metrics are collected
+const billingHour = 6
+
```

Comment:
I checked the old code and it did exactly the same, but could someone explain to me why 6 am and add the explanation in a comment? :sweat_smile:

Reply:
It probably has to do with exoscale how they update the data. @Kidswiss probably knows more.

Reply:
I assume the same but I don't know it either. Didn't find it on their website with a very quick search, too. 
Judging by the schedule (`'10 10,16,20 * * *'`) we still crawl it multiple times a day though, just setting the same billing hour to make sure we don't charge too much and just take the latest number.